judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/data/primitiveTypedElement.etl

```diff
@@ -152,3 +152,66 @@ rule CreateReadsDataPropertyForMappedTransferObjectTranferFieldDeclaration
+
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateDefaultDataExpressionTypeForMappedTransferObjectConstructor");
+        t.expression = expressionUtils.getJqlForExpression(s.rightValue, entityNamePrefix, entityNamePostfix);
+        s.equivalent("CreateDefaultDataPropertyForMappedTransferObjectConstructor").getterExpression = t;
+
+        log.debug("Created DataExpressionType for Transfer Object Default Value: " + t.expression);
+    }
+
+@greedy
+rule CreateDefaultDataPropertyForMappedTransferObjectConstructor
+    transform s : JSL!TransferDefault
+    to t: JUDOPSM!DataProperty {
+        guard: s.field.reference.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.field.reference.eContainer.map.isDefined()
+
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateDefaultDataPropertyForMappedTransferObjectConstructor");
+        t.name = defaultDefaultNamePrefix + s.field.reference.name + defaultDefaultNameMidfix + s.field.reference.eContainer.name + defaultDefaultNamePostfix;
+        t.dataType = s.field.reference.referenceType.getPSMEquivalent();
+        t.required = false;
+
+
+        s.field.reference.eContainer.map.entity.getPSMEquivalent().dataProperties.add(t);
+
+        log.debug("Created DataProperty for Transfer Object Default Value: " + s);
+    }
+
+@greedy
+rule CreateDefaultDataExpressionTypeForUnmappedTransferObjectConstructor
+    transform s : JSL!TransferDefault
+    to t: JUDOPSM!DataExpressionType {
+        guard: s.field.reference.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.field.reference.eContainer.map.isUndefined()
+
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateDefaultDataExpressionTypeForUnmappedTransferObjectConstructor");
+        t.expression = expressionUtils.getJqlForExpression(s.rightValue, entityNamePrefix, entityNamePostfix);
+        s.equivalent("CreateDefaultStaticDataForUnmappedTransferObjectConstructor").getterExpression = t;
+
+        log.debug("Created DataExpressionType for Transfer Object Default Value: " + t.expression);
+    }
+
+@greedy
+rule CreateDefaultStaticDataForUnmappedTransferObjectConstructor
+    transform s : JSL!TransferDefault
+    to t: JUDOPSM!StaticData {
+        guard: s.field.reference.referenceType.isKindOf(JSL!PrimitiveDeclaration) and s.field.reference.eContainer.map.isUndefined()
+
+        t.setId("(jsl/" + s.field.reference.getId() + ")/CreateDefaultStaticDataForUnmappedTransferObjectConstructor");
+        t.name = defaultDefaultNamePrefix + s.field.reference.name + defaultDefaultNameMidfix + s.field.reference.eContainer.name + defaultDefaultNamePostfix;
+        t.dataType = s.field.reference.referenceType.getPSMEquivalent();
+        t.required = false;
+
+        s.eContainer.eContainer.eContainer.getModelRoot().elements.add(t);
+        log.debug("Created StaticData for Transfer Object Default Value: " + s);
+    }
+    
```

Comment:
Hah!

Reply:
newline EOF (github does not mark missing newline EOF if not in "Files" view)

Reply:
Gabor's replace will fox this issue.